nixnet/database/_lin_sched_entry.py

```diff
@@ -72,32 +121,89 @@ def frames(self, value):
 
     @property
     def name(self):
+        # type: () -> typing.Text
+        """str: Get or set the name of the LIN schedule entry object.
+
+        Lowercase letters, uppercase letters, numbers,
+        and the underscore (_) are valid characters for the short name.
+        The space ( ), period (.), and other special characters are not supported within the name.
+        The short name must begin with a letter (uppercase or lowercase) or underscore, and not a number.
+        The short name is limited to 128 characters.
```

Comment:
It feels odd to me that this comment lists all the error cases but doesn't tell me what exception gets raised when one of them happens

Reply:
This is a really good point, and it's a precedent that permeates every C wrapped call. The precedent is that C API calls which return a nxStatus_t error code and are handled by _errors.check_for_error and are covered in docstrings by the C API documentation. For python documentation, we only cover deviations introduced in the python API. Adding driver errors to the python documentation would be a huge effort that would introduce a massively burdensome coupling with very little benefit.

Reply:
That makes sense.  Maybe in the future it would be useful to mention that in errors.py  or to handle some error in one of the examples to further document it.